Monika After Story/game/chess.rpy

```diff
@@ -179,7 +179,9 @@ init 1 python in mas_chess:
         "Tch-",
         "Winning isn't everything, you know...",
         "Ahaha,{w} I was just letting you win since you keep losing so much.",
-        "Oh, you won.{w} I should have taken this game seriously, then."
+        "Oh, you won.{w} I should have taken this game seriously, then.",
+        "I was just easy on you.",
+        "Did you used a{w} 'Solver'{w} or you're cheating?"
```

Comment:
Replace "used" with "use". 

Did you mean to put waits on both sides of "'Solver'"?

Also, that's pretty mean for her to accuse you of cheating just when you win.